### PR TITLE
R plugin

### PR DESCRIPTION
**Pull request by scottdraves**

### CODE REVIEW

File: plugin/r/src/main/java/com/twosigma/beaker/r/rest/RShellRest.java [hunk 1]
```diff
@@ -274,17 +266,16 @@ public SimpleEvaluationObject evaluate(@FormParam("shellID") String shellID,
...
+            java.nio.file.Files.deleteIfExists(p);
+        } catch (IOException e) {
             // ignore
-        //}
+        }
 
 	try {
             // direct graphical output
-            // con.eval("svg('" + file + "')"); // XXX removed temporarily
+            con.eval("svg('" + file + "')"); // XXX removed temporarily
```
Comment (anglee) [inline plugin/r/src/main/java/com/twosigma/beaker/r/rest/RShellRest.java]: Is the comment here outdated?

**Comment by anglee** [comment]
The file RShellRest.java has whitespace issues (using tabs instead of spaces). I didn't check other files. Please make sure they don't have whitespace issues either.

**Comment by anglee** [comment]
Is this just for cleaning up? Could you provide more context of what was driving these changes please?

File: plugin/r/src/main/java/com/twosigma/beaker/r/rest/RShellRest.java (hunk 1 above)
Comment (scottdraves) [inline plugin/r/src/main/java/com/twosigma/beaker/r/rest/RShellRest.java]: fixed

**Comment by scottdraves** [comment]
whitespace fixed

**Comment by scottdraves** [comment]
I don't understand your other comment.

**Pull request merged**